data/language/ko-KR.txt

```diff
@@ -1,4 +1,4 @@
-# STR_XXXX part is read and XXXX becomes the string id number.
+﻿# STR_XXXX part is read and XXXX becomes the string id number.
```

Comment:
You accidentally added the BOM here. Some editors automatically add this with UTF-8 encoded files.